torchci/scripts/updateQueueTimes.mjs

```diff
@@ -10,8 +10,8 @@ export function getS3Client() {
   return new S3Client({
     region: "us-east-1",
     credentials: {
-      accessKeyId: process.env.OUR_AWS_ACCESS_KEY_ID,
-      secretAccessKey: process.env.OUR_AWS_SECRET_ACCESS_KEY,
+      accessKeyId: process.env.AWS_ACCESS_KEY_ID,
```

Comment:
`OUR_*` is just our convention on Vercel because there is already a set of `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` there from the account owned by Vercel